src/pages/~/beta/[id].astro

```diff
@@ -4,16 +4,25 @@ import { firestore, getGame, getSession } from '../../../lib/game-saving/account
 import Editor from '../../../components/big-interactive-pages/editor'
 import StandardHead from '../../../components/standard-head.astro'
 import { signal } from '@preact/signals'
-import { RoomState, ConnectionStatus, type PersistenceState } from '../../../lib/state'
+import { RoomState, ConnectionStatus, type PersistenceState, PersistenceStateKind } from '../../../lib/state'
 import MobileUnsupported from '../../../components/popups-etc/mobile-unsupported'
 import { mobileUserAgent } from '../../../lib/utils/mobile'
 
+function convertStringToNumber(id:string){
+	let number = "";
+	for(let i = 0; i < id.length; i++){
+		number+=id.charCodeAt(i)
+	}
+	return Number(number)
+}
+
 const session = await getSession(Astro.cookies)
 if (!session) return Astro.redirect(`/login?to=${Astro.request.url}`, 302)
 
 let checkRoom = false;
 const game = await getGame(Astro.params.id!)
 if (!game) return Astro.redirect('/404', 302)
+if(convertStringToNumber(session.user.id) % 10 == 0 || session.user.email == "development@hackclub.com") Astro.redirect(`/~/${game.id}`)
```

Comment:
This is duplicate code that should be shared between here and its above usage...